CONTRIBUTING.rst

```diff
@@ -79,24 +84,35 @@ can be contributed to scikit-image.
 
 3. To submit your contribution:
 
-   * Push your changes back to your fork on GitHub::
+   * Push your changes back to your fork on GitHub:
+
+     ::
+
+       git push codemonkey transform-speedups
 
-      git push origin transform-speedups
+     A message will be displayed with a URL to open in your browser to start a
+     new pull request. Open it and click the green button.
 
-   * Enter your GitHub username and password (repeat contributors or advanced
-     users can remove this step by `connecting to GitHub with SSH
-     <https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh>`_).
+4. To work on someone else's PR:
 
-   * Go to GitHub. The new branch will show up with a green "pull request"
-     button -- click it.
+   ::
 
-   * If you want, post on the `developer forum
-     <https://discuss.scientific-python.org/c/contributor/skimage>`_ to explain your changes or
-     to ask for review.
+     git remote add friend-username git@github.com:friend-username/scikit-image
+     git fetch friend-username
+     git checkout pr-branch-name
+
+   Make changes, commit them, and then push back to the other person's
+   branch:
+
+   ::
+
+     git push friend-username pr-branch-name
 
 For a more detailed discussion, read these :doc:`detailed documents
 <../gitwash/index>` on how to use Git with ``scikit-image`` (:ref:`using-git`).
 
+If you get stuck, reach out to us on `Zulip <https://skimage.zulipchat.com/>`__.
```

Comment:
Might be a good to highlight this as a friendly tip:

```suggestion
.. tip:: 

   If you get stuck, reach out to us on 
   `our Zulip chat <https://skimage.zulipchat.com/>`__.
```
(Also suggesting a more expressive hyperlink title here.)